boms/cloud-lts-bom/RELEASING.md

```diff
@@ -29,3 +29,48 @@ The Rapid workflow uploads the artifact to OSSRH staging repository.
 [Instructions for releasing from OSSRH are on the internal team 
 site](https://g3doc.corp.google.com/company/teams/cloud-java/tools/developers/releasing.md#verify-and-release).
 
+
+## Making changes for a patch release
+
+Changes for a patch release should be committed to LTS release branches (not the master branch).
+
+We maintain multiple branches for the LTS BOM releases.
+Let's call the branches _LTS release branches_.
+An LTS release branch has a name "N.0.0-lts", where N is the major release version number,
+such as `1.0.0-lts` and `2.0.0-lts`.
+In this GitHub repository, the branches are [configured as protected](
+https://github.com/GoogleCloudPlatform/cloud-opensource-java/settings/branches),
+so that we do not accidentally push changes into them.
+
+Any changes for a patch release to the LTS BOM should be merged to one of the LTS release branches.
+This practice enables us to release a patch version of one of the old versions of the BOM,
+without using the master branch (or _HEAD_).
+
+Note that even when you prepare a patch release with higher patch number than 1,
+you commit changes to `N.0.0-lts` branch (the patch part is "0").
```

Comment:
Yeah as Dan G. mentioned in chat, it might be more clear to name these branches like `N.0.x-lts` instead of `N.0.0-lts` to indicate that the branch includes all the changes in the minor version range.

We used a similar naming practice in spring-cloud-gcp: https://github.com/spring-cloud/spring-cloud-gcp/branches
i.e. branches named 1.0.x and 1.1.x